src/components/dashboard/sidebar/utils.js

```diff
@@ -7,11 +7,13 @@ export const getOfferExpiringFirst = (offers) => {
     .sort((a, b) => new Date(a.endDatetime) - new Date(b.endDatetime))[0];
 };
 
-export const getPolicyExpiringFirst = (policies) => {
-  if (!policies) {
+export const getPolicyExpiringFirst = ({ expiredPolicies, unexpiredPolicies }) => {
+  if (!expiredPolicies || !unexpiredPolicies) {
     return undefined;
   }
-  return policies
-    .filter(policy => policy.active)
-    .sort((a, b) => new Date(a.subsidyExpirationDate) - new Date(b.subsidyExpirationDate))[0];
+  const hasUnexpiredPolicies = unexpiredPolicies.length > 0;
+  if (hasUnexpiredPolicies) {
+    return unexpiredPolicies.sort((a, b) => new Date(a.subsidyExpirationDate) - new Date(b.subsidyExpirationDate))[0];
+  }
+  return expiredPolicies.sort((a, b) => new Date(b.subsidyExpirationDate) - new Date(a.subsidyExpirationDate))[0];
```

Comment:
Otherwise, we return the expired policies displaying the last expired policies metadata.